sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_options.hpp

```diff
@@ -661,13 +652,10 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     Azure::Core::Context Context;
 
     /**
-     * @brief When deleting a directory, the number of paths that are deleted with each invocation
-     *        is limited. If the number of paths to be deleted exceeds this limit, a continuation
-     *        token is returned in this response header.  When a continuation token is returned in
-     *        the response, it must be specified in a subsequent invocation of the delete operation
-     *        to continue deleting the directory.
+     * @brief If "true", all paths beneath the directory will be deleted. If "false" and the
+     * directory is non-empty, an error occurs.
      */
-    Azure::Core::Nullable<std::string> ContinuationToken;
+    bool Recursive = false;
```

Comment:
I demand to revisit the default value

Reply:
Revisited with Jeffrey, will use the new design.